### PR TITLE
Fixing footer menu

### PR DESCRIPTION
**Pull request by webmaxru**
Footer menu height reduced - some main content was cutted out

### CODE REVIEW

File: templates/_layouts/wrapper.njk [hunk 1]
```diff
@@ -156,7 +156,13 @@
           {{ collections.all | eleventyNavigation('Main') | eleventyNavigationToHtml | safe }}
         </nav>
         <div class="footer">
-          <p><a href="https://twitter.com/PwaSummit" target="_blank">@PWASummit</a></p>
+          <p>
+            <a href="https://twitter.com/PwaSummit" target="_blank">@PWASummit</a><br /><a
```
Comment (kenchris) [inline templates/_layouts/wrapper.njk:160]: `<br>` is fine - this is not XHTML :-)

File: templates/_layouts/wrapper.njk [hunk 2]
```diff
@@ -156,7 +156,13 @@
...
+              href="https://2021.pwasummit.org/"
+              target="_blank"
+              >PWASummit'21</a
```
Comment (kenchris) [inline templates/_layouts/wrapper.njk:163]: I would keep the > on the earlier line and move the whole </a> to the next line

**Review by kenchris — APPROVED** [review]

**Comment by webmaxru** [comment]
@aarongustafson: with an update we lose our main CTA - "Submit CFP" on the mobile devices with smaller screen size:
image

My initial fix was exactly about that - to make sure the footer is as small as possible. Could you come up with design/implementation with CFP visible on iPhone SE?

**Comment by webmaxru** [comment]
I didn't check the details of the markup as it's just a quick copy/customize of 2021's edition :) But yes - the footer overlays main content. Which is completely fine if it's just 2 lines and becomes an issue on the smaller devices it the footer gets bigger

**Comment by aarongustafson** [comment]
I don’t know why that decision was made, but I’ll push an update to remove the sticky footer in a few minutes.

**Review by aarongustafson — APPROVED** [review]

**Pull request merged**